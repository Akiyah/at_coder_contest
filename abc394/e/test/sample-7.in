10
ba--------
-ba-------
--ba------
---ba-----
----ba----
-----ab---
------ab--
-------ab-
--------ab
---------a
