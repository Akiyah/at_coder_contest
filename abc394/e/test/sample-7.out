0 1 2 3 4 5 7 9 11 13
-1 0 1 2 3 4 6 8 10 12
-1 -1 0 1 2 3 5 7 9 10
-1 -1 -1 0 1 2 4 6 7 8
-1 -1 -1 -1 0 1 3 4 5 6
-1 -1 -1 -1 -1 0 1 2 3 4
-1 -1 -1 -1 -1 -1 0 1 2 3
-1 -1 -1 -1 -1 -1 -1 0 1 2
-1 -1 -1 -1 -1 -1 -1 -1 0 1
-1 -1 -1 -1 -1 -1 -1 -1 -1 0
